stroeer/web/article/v1/web_article.proto

```diff
@@ -19,6 +19,7 @@ message WebArticle {
   repeated stroeer.core.v1.Element elements = 8;
   repeated stroeer.core.v1.Author authors = 9;
   repeated stroeer.core.v1.BodyNode body_nodes = 10;
+  stroeer.core.v1.Metadata metadata = 11;
```

Comment:
With this change: why do we need `WebArticle` anymore? I guess its now 100% the same to `Article`. From a FE view this metadata object feels weird to me.

Reply:
For a start we'll add the detailed author-entities later on, which is not part of `Article`. There may be more to come, I fear.

Otherwise, where do you suggest putting those 4 dates? Protobuf leverages strongly typed APIs, so `fields.put('key', date.toString()` is not the right approach, imho. Should we could add another Container for dates instead?

Reply:
why do we need the Metadata in the WebArticle? I suppose its sufficient to have these in the Article for paperboy to respond with 404 or to filter unpulished elements?